### ISSUE TITLE
Create an org chart

### ISSUE THREAD

**Issue by iancrowther** [nodeschool/organizers#298]
Should we create an Org chart?

For example:

- Who do I speak to regarding ti.to
- Who do I speak to regarding our email london@nodeschool.io

**Comment by maxogden** [comment]
IMO we shouldn't have a shared ti.to or email because it makes it difficult to manage (because its all centralized). My opinion is I think it would be easier for everyone if chapters had their own tito and email. 

However @brianloveswords controls the nodeschool.io DNS, and there are apparently 25 people on the /nodeschool ti.to account and it looks like if you get invited you can create events under /nodeschool

**Comment by iancrowther** [comment]
@brianloveswords - what are your thoughts on the admin?

**Comment by SomeoneWeird** [comment]
Yeah, I think separate ti.to accounts for chapters is probably a good idea, makes everything easier to manage because you don't have to wait for a person who has admin to do stuff.

**Comment by iancrowther** [comment]
so i agree on the ti.to front.. 

what about email?

**Comment by maxogden** [comment]
I don't know an easy way to run email for a group. I've done Google Apps For Your Domain in the past but that requires a similar additional centralized management layer. If someone has an idea on how to set up nodeschool emails without a central bottleneck then that would solve it, otherwise I'm not sure how to make it maintainable

**Comment by martinheidegger** [comment]
Nodeschoolbot?

**Comment by brianloveswords** [comment]
@iancrowther @maxogden The main issues for me with Google Apps for us is scale. It costs $5/mo per email address. I'm fine to eat the cost for a few addresses, but if we're talking an email for every org, that becomes more problematic.

**Comment by RichardLitt** [comment]
We could host our own mail server, and not use Gmail at all. That should allow us to have somewhat limitless mailboxes, with aliases for each one; one per org shouldn't be a problem.

**Comment by mikeal** [comment]
You should do what Node.js is doing now.

https://github.com/nodejs/email

Basically, @rvagg wrote some code to configure [mailgun](http://www.mailgun.com/) and we point the DNS for email there. Then there's just a big json config in a repo that people can send pull requests to in order to get listed :)

**Comment by SomeoneWeird** [comment]
@brianloveswords Yeah, we really don't want to do that :)
@RichardLitt highly advise against running our own mail server
@mikeal that is v. cool :)

**Comment by brianloveswords** [comment]
Oh this is awesome @mikeal. I'm gonna try to set this up.

**Comment by brianloveswords** [comment]
Okay @iancrowther @maxogden I think [nodeschool/email](https://github.com/nodeschool/email) solves the email problem in a sufficiently decentralized way.

Steps to add an email address:
1. Requester submits a pull request that adds an alias to `nodeschool/aliases.json`
2. [Core team member](https://github.com/orgs/nodeschool/teams/core) ensures pull request passes Travis (json linting).
3. Core team member merges pull request.

Anything that hits the `master` branch will be automatically deployed.

We can send *10,000* free emails on the mailgun free account. I'll raise a new thread if we start getting close to that with legitimate use.

**Comment by SomeoneWeird** [comment]
I just enabled status check requirements for `master` on [nodeschool/email](https://github.com/nodeschool/email) - now travis _has_ to pass, before you can merge stuff :)

**Comment by iancrowther** [comment]
@brianloveswords - feel free to delete london@nodeschool.io from google apps
@mikeal - danke

**Comment by iancrowther** [comment]
lol - so back to the org chart? 

I never actually got much insight into my original question.  I think it would be useful

**Comment by maxogden** [comment]
IMO ideally the org chart would be a small group of people (more than 1 to increase "hit by a bus factor", but small enough group to eliminate any bureaucracy, maybe 10?). The job of the small group of people would be to manage passwords and administrative duties that should hopefully be few and far between (like redeploying nodeschoolbot when it breaks).

The rest of the org chart would just be chapters :)

**Comment by iancrowther** [comment]
ah yeah, I have not thought to include the chapters.  I think thats overkill and becomes a "thing" to admin.

what tasks people oversea?

- email
- nodeschoolbot
- newsletter
- website ci
- tito (which I assume will just become a placeholder movinf forward)
- ...?

**Comment by RichardLitt** [comment]
Those sound about right to me. Can we start with those, and add to it as needed?

It doesn't seem like there is a burning need for an org chart, @iancrowther. Unless I am missing something, as this ticket is a bit stale.

**Comment by SomeoneWeird** [comment]
I don't see this as a pressing need.
